seed/seed.json

```diff
@@ -2082,6 +2061,7 @@
                     "BETA",
                     "RELEASE"
                 ],
+                "max_version": "129.*",
```

Comment:
Would be great to set this to the version just before the change was merged. Thanks